DOCUMENTATION.md

```diff
@@ -2933,59 +2933,7 @@ Please visit the [Giraffe.ViewEngine](https://github.com/giraffe-fsharp/Giraffe.
 
 ### JSON
 
-By default Giraffe uses [Newtonsoft.Json](https://www.newtonsoft.com/json) for (de-)serializing JSON content. An application can modify the default serializer by registering a new dependency which implements the `Json.ISerializer` interface during application startup.
-
-Customizing Giraffe's JSON serialization can either happen via providing a custom object of `JsonSerializerSettings` when instantiating the default `NewtonsoftJson.Serializer` or by swapping in an entire different JSON library by creating a new class which implements the `Json.ISerializer` interface.
-
-By default Giraffe offers three `Json.ISerializer` implementations out of the box:
-
-| Name | Description | Default |
-| :--- | :---------- | :------ |
-| `NewtonsoftJson.Serializer` | Uses `Newtonsoft.Json` aka Json.NET for JSON (de-)serialization in Giraffe. It is the most downloaded library on NuGet, battle tested by millions of users and has great support for F# data types. Use this json serializer for maximum compatibility and easy adoption. | True |
-| `SystemTextJson.Serializer` | Uses `System.Text.Json` for JSON (de-)serialization in Giraffe. `System.Text.Json` is a high performance serialization library, and aims to be the serialization library of choice for ASP.NET Core. For better support of F# types with `System.Text.Json`, look at [FSharp.SystemTextJson](https://github.com/Tarmil/FSharp.SystemTextJson). | False |
-
-To use `SystemTextJson.Serializer` instead of `NewtonsoftJson.Serializer`, register a new dependency of type `Json.ISerializer` during application configuration:
-
-```fsharp
-let configureServices (services : IServiceCollection) =
-    // First register all default Giraffe dependencies
-    services.AddGiraffe() |> ignore
-
-    let serializationOptions = SystemTextJson.Serializer.DefaultOptions
-    // Optionally use `FSharp.SystemTextJson` (requires `FSharp.SystemTextJson` package reference)
-    serializationOptions.Converters.Add(JsonFSharpConverter(JsonUnionEncoding.FSharpLuLike))
-    // Now register SystemTextJson.Serializer
-    services.AddSingleton<Json.ISerializer>(SystemTextJson.Serializer(serializationOptions)) |> ignore
-```
-
-#### Customizing JsonSerializerSettings
```

Comment:
Should we really remove this section? How do we customize the JsonSerializerSettings now?

Reply:
Valid point, I've added it back but now below the "Using a different JSON serializer" section.